### PR TITLE
no more free event tickets for tour of taiwan

### PR DESCRIPTION
**Pull request by wyland1**
^

### CODE REVIEW

File: src/pages/membership.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ function MembershipPage(props: Props) {
                             </Text>
                         </li>
                         <li>
-                            <Text>Free Event Tickets to Tour of Taiwan and Night Market</Text>
+                            <Text>Free Event Tickets to Night Market</Text>
```
Comment (ameliahsu) [inline src/pages/membership.tsx:87]: can we add a line that says free entry to tour of taiwan, also update that in the section that says "what is a tasa membership card?"

**Review by ameliahsu — CHANGES_REQUESTED** [review]
add free entry to tot as a benefit and remove bundle prices

File: src/pages/membership.tsx (hunk 1 above)
Comment (wyland1) [reply src/pages/membership.tsx:87]: we aren't doing free entry to tour of taiwan anymore. Only free raffles (decided at today's board meeting)

Comment (wyland1) [reply src/pages/membership.tsx:87]: gonna submit another pull request

**Review by ameliahsu — APPROVED** [review]

**Pull request merged**